pkg/lib/oauth/handler/handler_authz.go

```diff
@@ -716,7 +716,7 @@ func (h *AuthorizationHandler) validateRequest(
 			return err
 		}
 	default:
-		return protocol.NewError("unsupported_response_type", "only 'code' response type is supported")
+		return protocol.NewError("unsupported response_type: %v", r.ResponseType())
```

Comment:
It should still be `unsupported_response_type` https://datatracker.ietf.org/doc/html/rfc6749#section-4.1.2.1 But we change the error description to `fmt.Sprintf("response_type %v is not supported")`

Reply:
Oh sorry it seems I deleted something wrong :sosad: